pkg/vif/device/device.go

```diff
@@ -1,4 +1,4 @@
-package vif
+package device
```

Comment:
I liked the name `vif.Device` a lot more than `device.Device` because it's clear and concise. `device.Device` stutter and says nothing about the type of the device.

I realize that the current package structure under `vif` is awkward. Static routing and the `routing` package isn't related to the `vif` for instance. But introducing stuttering like `device.Device` or `netstack.Stack` doesn't improve the situation. I also feel that the stack is a very integral part of the VIF so keeping it adjacent to the device is a good thing.

Suggestion:
Move the routing up directly under `pkg` to improve separation of concern. Keep your current split of the stack.go so that logging is a file of its own, but let them stay in the `vif` package. That way, the `vif` package is only the `Device`, the `Stack`, and the private `dlogEmitter` for the `Stack`. The stutter is gone, and the names are more descriptive.

This will also allow for the `InitLogger` to be unexported and called just before the stack is created in `NewTunnelVIF`, driving the encapsulation a step further.